app/views/posts/index.html.erb

```diff
@@ -1 +1,10 @@
-<p><%= @posts %></p>
\ No newline at end of file
+<%= render "/users/user" %>
+<%@posts.each do |post| %>
+<p>Post# <%=post.id%></p>
+<p><%= post.text %><p>
+Comments: <%= post.comments_counter%>
+Likes: <%= post.likes_counter %>
+<br>
+<%= render partial:"comment", :locals => {:specific_post => post} %>
+<%end%>
+<button>Pagination</button>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes), Please add basic styling to posts index view to make it match the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png).